UnityProject/Assets/Scripts/Equipment/Equipment.cs

```diff
@@ -214,7 +214,7 @@ public IEnumerator SetPlayerLoadOuts()
 				}
 				else if (!string.IsNullOrEmpty(gearItem.Value))
 				{
-					Debug.Log(gearItem.Value + " creation not implemented yet.");
+//					Debug.Log(gearItem.Value + " creation not implemented yet.");

```

Comment:
Yeah thats annoying...
Maybe one day we should make a global "Verbose" flag to turn all those verbose outputs on and off.

Reply:
Single verbose flags don't cut it. You'll want categorized logging and stuff.

Reply:
STFU PJB.
I know you're a 17 year old cunt, so go back to mommy to complain about her brainfarts instead of mine.